### PR TITLE
FIX MongoDB supported versions information in documentation

### PR DESCRIPTION
**Pull request by fgalan**
Related with #1714 

@Madhu-NEC could you have a look and tell us if this PR is ok with you? Thanks!

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md [hunk 1]
```diff
@@ -392,8 +392,9 @@ Despite the old encoding will be deprecated in the future, it is possible to swi
 #### <a name="section2.3.4"></a>About supported versions of MongoDB
 This sink has been tested with the following versions of Mongo:
 
-* 3.2.6
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md]: 3.2.6 has been removed as it has passed End-of-Life according https://www.mongodb.com/support-policy

**Comment by Madhu-NEC** [comment]
Hi @fgalan ,

Thanks for raising the PR.

I have a small query. The mongodb driver version mentioned in [pom.xml](https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-common/pom.xml#L108) is 3.0.4. I have found a [document](https://docs.mongodb.com/ecosystem/drivers/java/#mongodb-compatibility) where I found the compatibility of mongoDB versions with driver versions. 
Should we change the mongodb driver version in pom.xml as well? **Please share your opinion on this.**

Changes are also required at https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-common/installation_and_administration_guide/install_from_sources.md for mongodb driver.

**Note:** When I was testing with MongoDB v4.2, I didn't change pom.xml (i.e. mongodb driver version)

**Comment by fgalan** [comment]
Driver upgraded in commit 3055801

**Comment by Madhu-NEC** [comment]
Thanks @fgalan 
It looks good to me now.
Could you please merge it?

**Comment by Madhu-NEC** [comment]
Hi @fgalan ,
If there is no issue, could you please merge it?

**Pull request merged**